components/github/sources/new-repository/new-repository.js

```diff
@@ -1,32 +1,41 @@
-const github = require("../../github.app.js");
+const common = require("../common-polling.js");
```

Comment:
We could set up an organization webhook to watch an organization for new repositories. However, doing it this way through polling allows any user to check for new repositories in their account. If you think it would be better to require an organization prop and only watch for new repositories belonging to an organization, I'll make the change.

Reply:
Ahh I see, then yeah we can add support later for the use case you mention.